src/main/flight/gyroanalyse.c

```diff
@@ -112,13 +152,15 @@ void gyroDataAnalyseInit(uint32_t targetLooptimeUs)
     // otherwise we need to calculate a FFT sample frequency to ensure we get 3 samples (gyro loops < 4K)
     const int gyroLoopRateHz = lrintf((1.0f / targetLooptimeUs) * 1e6f);
     
-    fftSamplingRateHz = MIN((gyroLoopRateHz / 3), fftSamplingRateHz);
+    fftSamplingRateHz = MIN((gyroLoopRateHz / 3), fftSamplingRateHz); // 1333hz at 8k medium
+
+    fftResolution = (float)fftSamplingRateHz / FFT_WINDOW_SIZE; // 41.65hz per bin for medium
 
-    fftResolution = (float)fftSamplingRateHz / FFT_WINDOW_SIZE;
+    fftStartBin = MAX(2, dynNotchMinHz / lrintf(fftResolution)); // can't use bin 0
```

Comment:
Why do you also leave out bin 1?

Reply:
Bin 1 is almost always still carrying control information.  I do compare bin 2 to bin 1 if bin 2 is max.  I would not want to compare bin 1 to bin 0, if bin 1 was max.  Setting the start bin to 2 was the easiest way I found to achieve this.  

Bin 0 is DC value of the entire FFT block, not a frequency bin.  When I logged it, it held erratic values that were useless for comparisons of AC amplitudes.

Reply:
Can't this `MAX()` just be removed to avoid confusion? It doesn't really matter at all because in [settings.c](https://github.com/betaflight/betaflight/blob/master/src/main/cli/settings.c#L659) there's a minimum 60Hz value anyways. With this resolution we have here 60Hz will fall to bin 1.